Sage-1545: Prep for cluster deployment.

seanshahkarami opened the pull request:
This PR starts to split Django's settings into dev / prod to allow local testing and also deployment to the production cluster. 

To simplify the number of different database configurations, I've also switch the docker compose environment to use mysql to match prod.

iperezx commented:
Some notes:
- README does not reflect the proposed changes
- https://github.com/waggle-sensor/beekeeper-manifest/blob/SAGE-1545-prep-for-deployment/.github/workflows/cicd.yml#L18 was never a great test, so I think it's just passing due to an error. I think you changed the endpoint with your changes?
- Any reason why you don't use the traditional make cmds? For example: `make all`, `make clean`, etc? (minor)
- Any reason why you took out the healthcheck?

seanshahkarami commented:
Thanks for the feedback. I make the following changes:
* Updated docs to match the setup I'm using here.
* Updated CI to actually run the migrations and test suite (currently empty in this branch..).
* Added a mysql health check. (I removed the Postgres one because I'd switched to using MySQL .)

For the make commands, I picked ones that I think match this situation where you want to deploy and then do some post deployment setup. (start, stop, test, migrate, collectstatic, etc)

I personally don't find the typical "compile oriented" make commands (build, test, clean, etc) very intuitive for this.
